bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -521,7 +521,11 @@ spec:
           export OVN_NB_DAEMON=$(ovn-nbctl --pidfile=/var/run/ovn/ovn-nbctl.pid \
             --detach \
             -p /ovn-cert/tls.key -c /ovn-cert/tls.crt -C /ovn-ca/ca-bundle.crt \
-            --db "{{.OVN_NB_DB_LIST}}")
+            --db "{{.OVN_NB_DB_LIST}} --log-file=/run/ovn/ovn-nbctl.log")
+
+          # include nbctl daemon logging, allow for ovn-nbctl to create the log file
+          sleep 1
+          tail -f /run/ovn/ovn-nbctl.log &
```

Comment:
@trozet `tail -f --retry` instead of a sleep 1?

Reply:
oh never knew about that argument, nice I'll change it